.gitignore

```diff
@@ -140,3 +140,4 @@ ehthumbs.db
 Thumbs.db
 
 *.user.2.7pre1
+*.py
```

Comment:
please remove this addition. Adding a python script to this repo is not outside the realm of possibilities, so I don't think it should be added to the gitignore